aiofreepybox/aiofreepybox.py

```diff
@@ -13,6 +13,7 @@
 import aiofreepybox
 from aiofreepybox.exceptions import *
 from aiofreepybox.access import Access
+from aiofreepybox.api.tv import Tv
```

Comment:
You also need to instantiate the `Tv` module in `open`

Reply:
fixed